src/clients/circleci-private/me.ts

```diff
@@ -58,10 +68,29 @@ export class MeAPI {
       const result = FollowedProjectResponseSchema.parse(rawResult);
 
       pageCount++;
-      allProjects.push(...result.items);
+
+      // Only add projects that haven't been seen before
+      for (const project of result.items) {
+        if (!seenSlugs.has(project.slug)) {
+          seenSlugs.add(project.slug);
```

Comment:
why not just make allProjects a dict? then you dont need seenSlugs?

Reply:
good catch - i may be able to just remove this logic entirely, it was leftover from when i was having some duplication issues, which i think was related to a bug in my pagination (fixed now). so i'll check on that, and either remove this or switch to a dict approach

Reply:
removing - no longer needed